pkg/reconciler/common/transformer_additional_options.go

```diff
@@ -135,6 +135,60 @@ func (ot *OptionsTransformer) updateAnnotations(u *unstructured.Unstructured, an
 	return nil
 }
 
+func (ot *OptionsTransformer) updatePodTemplateLabels(u *unstructured.Unstructured, labels map[string]string) error {
```

Comment:
Had to go with `NestedMap`, `RemoveNestedField` and `SetNestedMap`  here given there's no method for pod templates in https://github.com/tektoncd/operator/blob/main/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go